app/locales/en.json

```diff
@@ -183,6 +183,7 @@
     "launch_screen3_header_location": "Help contain the spread of the virus and protect others in your community.",
     "launch_screen3_subheader_location": "If you are diagnosed, you can choose to share your location diary with your local Health Department so they can anonymously inform others who may have crossed paths with the virus.",
     "launch_screen4_header_location": "Be notified when your PathCheck app estimates that you may have crossed paths with the virus.",
+    "launch_screen4_subheader_location": "Learn the dates of possible exposures and see daily cumulative possible exposure time.",
```

Comment:
Also fixed in https://github.com/Path-Check/covid-safe-paths/pull/1334 -- lets ensure we don't have duplicate keys